### PR TITLE
S1t5data presentation

### PR DESCRIPTION
**Pull request by cwoodwarduk**
Review front end output presentation and tests for functions.

### CODE REVIEW

File: calumsCollection.sql [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+
+# Dump of table myBikes
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `myBikes`;
+
+CREATE TABLE `myBikes` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `manufacturer` varchar(50) DEFAULT NULL,
```
Comment (charliecog) [inline calumsCollection.sql]: allow a larger length eg: 255

File: calumsCollection.sql [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  `model` varchar(50) DEFAULT NULL,
+  `wheelset` varchar(70) DEFAULT NULL,
+  `groupset` varchar(100) DEFAULT NULL,
+  `image` longblob,
```
Comment (charliecog) [inline calumsCollection.sql]: just use a varchar

File: functions.php [hunk 3]
```diff
@@ -35,17 +37,23 @@ function getData(PDO $db) :array {
  */
 function getBike(array $data) :string
```
Comment (charliecog) [inline functions.php]: param should be named $bikes

File: styles.css [hunk 4]
```diff
@@ -4,16 +4,67 @@
...
 body {
     font-size: 12px;
+    background-image: url("images/old-bicycle.jpg");
+    background-size: 100%;
+    background-position: top 70px;
 }
 
 h1 {
     font-family: 'Permanent Marker', cursive;
     font-size: 3rem;
```
Comment (charliecog) [inline styles.css]: if you use rems you should manually set the font size on the html element other wise its a lottery what will happen

File: styles.css [hunk 5]
```diff
@@ -4,16 +4,67 @@
...
     font-size: 1.5rem;
 }
 
+section {
+    margin: 0 auto;
+}
+
+.tableHeadings {
+    overflow: auto;
+    margin: 0 auto;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 6]
```diff
@@ -4,16 +4,67 @@
...
+    border-bottom: 2px solid black;
+    background-color: #eaeffa;
+}
+
+.tableHeadings div {
+    float: left;
+    padding-left: 8px;
+    width: 20%;
+    font-family: 'M PLUS 1p', sans-serif;
```
Comment (charliecog) [inline styles.css:40]: not doing anything

File: styles.css [hunk 7]
```diff
@@ -4,16 +4,67 @@
...
+    font-size: 1.2rem;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 8]
```diff
@@ -4,16 +4,67 @@
...
 
 h1 {
     font-family: 'Permanent Marker', cursive;
     font-size: 3rem;
     text-align: center;
+    margin: 70px;
 }
 
 h2 {
     font-family: 'M PLUS 1p', sans-serif;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 9]
```diff
@@ -4,16 +4,67 @@
...
+    float: left;
+    padding-left: 8px;
+    width: 20%;
+    font-family: 'M PLUS 1p', sans-serif;
+    font-size: 1.2rem;
+}
+
+.tableContent {
+    overflow: auto;
+    margin: 0 auto;
```
Comment (charliecog) [inline styles.css]: not doing anything

File: styles.css [hunk 10]
```diff
@@ -4,16 +4,67 @@
...
+}
+
+.tableContent div {
+    float: left;
+    padding-left: 8px;
+    width: 20%;
+    height: 170px;
+    font-family: 'M PLUS 1p', sans-serif;
+    font-size: 1.1rem;
+    border-bottom: 1px solid darkgrey;
```
Comment (charliecog) [inline styles.css]: use a hex to stay consistent

**Review by charliecog — COMMENTED** [review]
Couple of bits, good start though. Have a go at responsiveness!

File: styles.css (hunk 5 above)
Comment (cwoodwarduk) [reply styles.css]: Removed

File: styles.css (hunk 6 above)
Comment (cwoodwarduk) [reply styles.css:40]: Removed

File: styles.css (hunk 7 above)
Comment (cwoodwarduk) [reply styles.css]: Removed

File: styles.css (hunk 9 above)
Comment (cwoodwarduk) [reply styles.css]: Removed

File: styles.css (hunk 10 above)
Comment (cwoodwarduk) [reply styles.css]: Changed to hex.

File: calumsCollection.sql (hunk 1 above)
Comment (cwoodwarduk) [reply calumsCollection.sql]: Done

File: calumsCollection.sql (hunk 2 above)
Comment (cwoodwarduk) [reply calumsCollection.sql]: Done

File: functions.php (hunk 3 above)
Comment (cwoodwarduk) [reply functions.php]: Job done.

File: styles.css (hunk 4 above)
Comment (cwoodwarduk) [reply styles.css]: Sorry about that. All done!

File: styles.css (hunk 8 above)
Comment (cwoodwarduk) [reply styles.css]: Changed

**Review by sudokufan — APPROVED** [review]
👍🏻

**Pull request merged**